.github/CODEOWNERS

```diff
@@ -16,48 +16,48 @@
 /docs/                                  @edenlightning @tchaton @borda @awaelchli @RobertLaurella
 /.github/*.md                           @edenlightning @williamfalcon @borda
 /.github/ISSUE_TEMPLATE/                @edenlightning @borda @tchaton
-/docs/source/conf.py                    @borda @awaelchli @carmocca
-/docs/source/index.rst                  @williamfalcon
-/docs/source/levels                     @williamfalcon @RobertLaurella
-/docs/source/expertise_levels           @williamfalcon @RobertLaurella
+/docs/source-PL/conf.py                    @borda @awaelchli @carmocca
```

Comment:
Why do we need this `-` separator instead of

`docs/pytorch_lightning/source/...`